tests/src/JIT/CodeGenBringUpTests/LocallocCnstB1.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+
+using System;
+using System.Runtime.CompilerServices;
+public class BringUpTest
+{
+    const int Pass = 100;
+    const int Fail = -1;
+
+    // Reduce all values to byte
+    public static unsafe bool CHECK(byte check, byte expected) {
+        return check == expected;
+    }
+
+    [MethodImplAttribute(MethodImplOptions.NoInlining)]
+    public static unsafe int LocallocCnstB1()
+    {
+        byte* a = stackalloc byte[1];
+        for (int i = 0; i < 1; i++)
+        {
+            a[i] = (byte) i;
+        }
+
+        for (int i = 0; i < 1; i++)
+        {
+            if (!CHECK(a[i], (byte) i)) return i;
+        }
+
+        return -1;
+    }
+
```

Comment:
Please verify that in a C# release compilation (e.g., /o+), that the JIT actually does generate a localloc. I'm worried that a really smart compiler could figure out what is going on here and optimize the entire function away.